modules/reference/pages/configuration-parameters.adoc

```diff
@@ -1530,4 +1530,13 @@ a|
 * The default value is 0. When the value is 0, the threshold is dynamically decided by TigerGraph, based on current memory usage statistics.
 | `TransactionSizeLimit=1073741824`
 
+|GSE
+| `ServFromDisk`,
+GSE.BasicConfig.Env
+a|
+* Enable the disk mode to read vertex ID from the disk instead of memory by setting it to `1` (Default = `0`). This will help reduce the GSE startup time significantly by skipping loading vertices into the memory from the disk with the trade-off of slight (< 10%) performance degradation for data loading and query processing.
```

Comment:
I am not sure about the < 10% performance degradation, which might be true for query processing, but I am afraid the impact might be larger for data loading.

Reply:
I asked Yuanyang some questions about the perf test method to justify this. Once I confirm with him, I can give more details here. Or I can just ditch the number and just keep the wording *slight*. The degradation actually can vary by the data size.

Reply:
Let's confirm with Yuanyang first, I am afraid the regression could be 30% or higher.

Reply:
@TanYongSql @junhyungshin-tg What's the resolution to the performance slowdown?